Nuno-Ferreira/File_Renaming_APP
# 14/Script/Get-variables-from-user-input

## Description
Adds the `rename_files` script that takes in the arguments from the user input in the GUI.
Closes #14 

## Test Plan
- Run `python src/gui.py`
- Choose a folder and insert a name sequence
- Click `Rename` 
- Check the console and print statements align with the user input
